### DISCUSSION TITLE
Discussion on Switching from Adminer Due to DEPRECATION and Lack of Maintenance

### DISCUSSION THREAD

**Discussion by javadzarezadeh** [fastapi/full-stack-fastapi-template#1399] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
N/A
```

### Description

Dear Sebastián,

It appears that Adminer is no longer maintained and has been marked as [DEPRECATED](https://hub.docker.com/_/adminer). I'd like to initiate a discussion with the FastAPI community, particularly with you, about whether we should consider switching to an alternative or continue using it for now. There's a fork called [AdminerEvo](https://github.com/adminerevo/adminerevo) that might be a viable option, although it currently lacks an official Docker image.

Issue on the official Adminer docker: [here](https://github.com/TimWolla/docker-adminer/issues/147)

Thank you all for participating!

### Operating System

macOS

### Operating System Details

N/A

### Python Version

N/A

**Comment by javadzarezadeh** [comment]
I found a relatively new database IDE called [WebDB](https://github.com/WebDB-App/app), which also has a [Docker image](https://docs.webdb.app/installation/compose). It looks quite modern! Another option could be pgAdmin 4, which also has a [Docker image](https://www.pgadmin.org/download/pgadmin-4-container/) and even utilizes [Traefik](https://www.pgadmin.org/docs/pgadmin4/latest/container_deployment.html#traefik), but it only supports PostgreSQL.

**Comment by YuriiMotov** [comment, marked as answer]
Adminer seems to be alive again
